CommonLibSF/include/RE/A/Actor.h

```diff
@@ -19,6 +19,12 @@ namespace RE
 			return *singleton;
 		}
 
+        inline static bool IsPlayerCharacterInChargen()
```

Comment:
`InCharge`?

Reply:
`ChargenMenu`, I believe

Reply:
Chargen is the status before PC first enters Character creation menu. For Skyrim I believe this will be true untill you exit the carriage.
I've always known it as "Chargen", the console command "SetInChargen" persists in Beth games at least from FO3.
Perhaps renaming it to "IsPlayerCharacterInChargenMode" would be more understandable?

Reply:
No worries. Wasn't sure of the term.